lang/cc65/Portfile

```
@@ -1,8 +1,10 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4; -*-
+# vim: set fileencoding=utf-8 tabstop=4 shiftwidth=4 softtabstop=4 noexpandtab filetype=tcl :
 PortSystem 1.0
 
```

Use the github portgroup:

```suggestion
PortGroup       github 1.0

```

---

done!